src/include/lean/lean.h

```diff
@@ -1981,6 +1993,429 @@ static inline uint8_t lean_int8_dec_le(uint8_t a1, uint8_t a2) {
     return lhs <= rhs;
 }
 
+/* Int8 -> other */
+static inline uint16_t lean_int8_to_int16(uint8_t a) { return (uint16_t)(int16_t)(int8_t)a; }
+static inline uint32_t lean_int8_to_int32(uint8_t a) { return (uint32_t)(int32_t)(int8_t)a; }
+static inline uint64_t lean_int8_to_int64(uint8_t a) { return (uint64_t)(int64_t)(int8_t)a; }
+
+
+/* Int16 */
+LEAN_EXPORT int16_t lean_int16_of_big_int(b_lean_obj_arg a);
+static inline uint16_t lean_int16_of_int(b_lean_obj_arg a) {
+    int16_t res;
+
+    if (lean_is_scalar(a)) {
+        res = (int16_t)lean_scalar_to_int64(a);
+    } else {
+        res = lean_int16_of_big_int(a);
+    }
+
+    return (uint16_t)res;
+}
+
+static inline uint16_t lean_int16_of_nat(b_lean_obj_arg a) {
+    int16_t res;
+
+    if (lean_is_scalar(a)) {
+        res = (int16_t)lean_unbox(a);
+    } else {
+        res = lean_int16_of_big_int(a);
+    }
+
+    return (uint16_t)res;
+}
+
+static inline lean_obj_res lean_int16_to_int(uint16_t a) {
+    int16_t arg = (int16_t)a;
+    return lean_int64_to_int((int64_t)arg);
+}
+
+static inline uint16_t lean_int16_neg(uint16_t a) {
+    int16_t arg = (int16_t)a;
+
+    return (uint16_t)(-arg);
+}
+
+static inline uint16_t lean_int16_add(uint16_t a1, uint16_t a2) {
+    int16_t lhs = (int16_t) a1;
```

Comment:
You have some inconsistency about whether you have a space or not in the cast, could you clean that up?